CONTRIBUTING.md

```diff
@@ -62,63 +103,90 @@ to follow step 3 and beyond:
+A commit of the type feat introduces a new feature to the codebase
+(this correlates with MINOR in semantic versioning).
+
+e.g.:
 
-eg: 
 ```
 feat: xxxxxxxxxx
-````
+```
 
 A commit of the type fix patches a bug in your codebase (this correlates with PATCH in semantic versioning).
 
-eg: 
+e.g.:
+
 ```
 fix: xxxxxxxxxx
-````
+```
+
+Commits types such as as `docs:`,`style:`,`refactor:`,`perf:`,`test:`
+and `chore:` are valid but have no effect on versioning. **It would be great
+if you use them.**
+
+Use `npm run commitmsg` to check your commit message format.
+
+**PRs that do not follow the commit message guidelines will not be merged.**
 
-Commits types as `docs:`,`style:`,`refactor:`,`perf:`,`test:` and `chore:` are valid but has no effect on versioning, but, it would be great if you use them.
+## Update Tests
 
-Use `npm run commitmsg` to check your commit message.
+**Any change in source code must include test updates**.
 
-> All PR that does not follow the commit guidelines will be hold until messages are fixed. 
+If you need help with how testing works, please [refer to the following guide](https://github.com/verdaccio/verdaccio/wiki/Running-and-Debugging-tests).
 
-## Update Test
+**If you are introducing new features, you MUST include new tests. PRs for
+features without tests will not be merged.**
 
-Any change in the sour code **must to include test update**, if you need support about how test works, please [refers to the following guide](https://github.com/verdaccio/verdaccio/wiki/Running-and-Debugging-tests). Please include test whether is a new feature, otherwise will be hold and never be merged.
+Things excluded from tests:
+* Documentation
+* Website
+* Build
+* Deployment
+* Assets
+* Flow types
 
-> Documentation, website, build, deployment, assets  or flow types are excluded in this section
+## Develop Plugins
 
-## Plugins
+Plugins are add-ons that extend the functionality of the application.
 
-Plugins are Add-ons that extend the functionality of the application. Whether you want develop your own plugin I'd suggest do the following:
+If you want to develop your own plugin:
 
-1. Check whether there is a legacy sinopia plugin for the feature that you need at [npmjs](https://www.npmjs.com/search?q=sinopia).
-2. There is a [life-cycle to load a plugin](https://github.com/verdaccio/verdaccio/blob/master/lib/plugin-loader.js#L22) you should keep on mind.
-3. You are free to host your plugin in your repository, whether you want to host within in our organization, feel free to ask, we'll happy to host it.
-4. Try a describe widely your plugin to provide a deeply understanding to your users.
+1. Check whether there is a legacy Sinopia plugin for the feature that you need
+    via [npmjs](https://www.npmjs.com/search?q=sinopia)
+2. Keep in mind the [life-cycle to load a plugin](https://github.com/verdaccio/verdaccio/blob/master/lib/plugin-loader.js#L22)
```

Comment:
This link is broken, I think we should point to https://verdaccio.org/docs/en/dev-plugins instead